docs/src/pages/[platform]/getting-started/troubleshooting/troubleshooting.angular.mdx

```diff
@@ -1,72 +1,9 @@
-import { TabItem, Tabs } from '@aws-amplify/ui-react';
+import { Fragment } from '@/components/Fragment';
 
-### Global and Process Shim
-
-Angular 6+ does not include shims for `global` or `process`. Follow the instructions below to resolve `global` or `process` reference errors.
-
-<Tabs>
-<TabItem title="Angular 15+">
-
-First, create `src/polyfills.ts` and add the following:
-
-```ts
-(window as any).global = window;
-(window as any).process = {
-  env: { DEBUG: undefined },
-};
-```
-
-Then, open your `angular.json` file, and add `src/polyfills.ts` to `polyfills` array(s) in your `angular.json`. These  arrays are located in `projects.<project-name>.architect.<task-name>.options`.
-
-```json{3}
-"polyfills": [
-  "zone.js",
-  "src/polyfills.ts"
-],
-```
-
-And finally, make sure to add `src/polyfills` to `files` in your `tsconfig.app.json`:
-
-```json{4}
-{
-  "files": [
-    "src/main.ts",
-    "src/polyfills.ts"
-  ],
-}
-```
-
-  </TabItem>
-
-  <TabItem title="Angular 7-14">
-
-Add the following to your `src/polyfills.ts`:
-
-```ts
-(window as any).global = window;
-(window as any).process = {
-  env: { DEBUG: undefined },
-};
-```
-
-  </TabItem>
-</Tabs>
-
-
-### Unhandled Promise rejection console errors
-
-If you see `Unhandled Promise rejection: [message] ; zone: <root> ; ...` errors on console, you can add the following code after you import `zone.js` inside `src/polyfills.ts` file:
-
-```ts{3}
-import 'zone.js/dist/zone'; // Included with Angular CLI.
-
-(window as any).Zone['__zone_symbol__ignoreConsoleErrorUncaughtError'] = true;
-```
-
-### Typescript Strict Mode Errors
-
-Angular 12+ now turns on typescript strict mode by default. This has known type issues when installing Amplify. You can either work around this by disabling strict mode inside your `tsconfig.json`, or by following the TypeScript strict [troubleshooting guide](https://docs.amplify.aws/lib/troubleshooting/strict-mode/q/platform/js/).
+<Fragment>
+  {({ platform }) => import('./shared/angular-polyfill.mdx')}
+</Fragment>
 
 ### Warning: CommonJS or AMD dependencies
 
-If you see CommonJS or AMD dependencies optimization bailouts warnings on Angular 9+, add the following [content](https://gist.github.com/wlee221/6d98d96740bea6f53327b4db4a432616) to your `angular.json`.
+If you see CommonJS or AMD dependencies optimization bailouts warnings, add the following [content](https://gist.github.com/wlee221/6d98d96740bea6f53327b4db4a432616) to your `angular.json`.
```

Comment:
We should really move away from this personal gist. I created a ticket for that.